cdist/conf/type/__user_groups/explorer/oldusermod

```diff
@@ -22,7 +22,8 @@ os="$($__explorer/os)"
 
 if [ "$os" = "netbsd" ]; then
     echo netbsd
-    exit
+elif [ "$os" = "freebsd" ]; then
+    echo freebsd
+else
+    usermod --help | grep -q -- '-A group' && echo true || echo false
 fi
```

Comment:
Do we need the os = freebsd / netbsd here at all? If you catch on $os in gencode-remote, it looks like this value is not needed at all